tests/common/test_version.py

```diff
@@ -123,6 +124,23 @@ def test_get_lis_version_should_return_a_string(self):
         lis_version = get_lis_version()
         self.assertIsInstance(lis_version, str)
 
+    def test_get_daemon_version_should_return_the_version_that_was_previously_set(self):
+        set_daemon_version("1.2.3.4")
```

Comment:
This function's argument is named `flexible_version`. Should this be a `FlexibleVersion` object? If not, maybe the function argument should be renamed.

Reply:
thanks for pointing it out. i intended to make it a FlexibleVersion for symmetry with the getter. will update the code.

Reply:
done